tests/helpers.py

```diff
@@ -58,12 +58,12 @@ def trigger(
             event = getattr(ctx.on, event)(tuple(state.containers)[0])
         else:
             event = getattr(ctx.on, event)()
-    with ctx.manager(event, state=state) as mgr:
+    with ctx(event, state=state) as event:
         if pre_event:
```

Comment:
this is legacy machinery that is gone from the public API, at some point we should also clean it up from the testing infra